src/qt/psbtoperationsdialog.cpp

```diff
@@ -141,7 +141,7 @@ void PSBTOperationsDialog::saveTransaction() {
     filename_suggestion.append(".psbt");
     QString filename = GUIUtil::getSaveFileName(this,
         tr("Save Transaction Data"), filename_suggestion,
-        tr("Partially Signed Transaction (Binary) (*.psbt)"), &selected_filter);
+        tr("Partially Signed Transaction (Binary)") + QLatin1String(" (*.psbt)"), &selected_filter);
```

Comment:
Annotate the `tr` here and later too?

Reply:
```suggestion
        tr("Partially Signed Transaction (Binary)", "Name of binary PSBT file format") + QLatin1String(" (*.psbt)"), &selected_filter);
```

Reply:
Thanks! [Done](https://github.com/bitcoin-core/gui/pull/220#issuecomment-785840104).